AppCenterDistribute/AppCenterDistributeTests/MSDistributeConfigureTests.m

```diff
@@ -15,33 +15,32 @@ @interface MSDistributeConfigureTests : XCTestCase
 
 @implementation MSDistributeConfigureTests
 
-- (void)testConfigureFlagsBeforeStart {
+- (void)testDisableAutomaticCheckForUpdateBeforeStart {
 
   // If
-  [MSDistribute sharedInstance].distributeFlags = -1;
```

Comment:
Precondition should be `istribute.automaticCheckForUpdatesDisabled == NO` to verify the assert at the end of the test.